public/index.html

```diff
@@ -20,7 +20,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>cmd-f Portal</title>
+    <title>HackCamp Portal</title>
```

Comment:
nit: would export this to `utility/Constants.js@copyText`

Reply:
Am I able to do this or would I have to use an env variable?

Reply:
don't think you need an env var, just add a link to the object and then reference the object here :)) just makes changing things easier for the future

Reply:
Might be out of my element here, but have no idea how to do this :') Would you mind providing some code snippets on how to achieve this?

Reply:
I think the easiest would probably to remove the definition here and use `next/head` https://nextjs.org/docs/api-reference/next/head and a Custom `App` https://nextjs.org/docs/advanced-features/custom-app

BUT this is a lot of effort for not a lot of gain so u can probs ignore

Reply:
OH gotcha, yeah I think I'll get this out for the time being and look into doing this for the future. Good flag though 👍